docker-compose.yaml

```diff
@@ -79,15 +100,15 @@ services:
     image: content-watcher-service
     build:
       context: .
-      dockerfile: dev.Dockerfile
+      dockerfile: Dockerfile
       tags:
         - content-watcher-service:latest
-    env_file:
-      - .env.docker.dev
```

Comment:
nit: is it not preferred to use a file for env?

Reply:
In gateway, we moved the env files into the docker-compose and seems like it's easier to manage that way.